### PR TITLE
Travis CI for Aragon Wiki

### PR DESCRIPTION
**Pull request by antonyip**
github-token is Generated in Github Developer Settings -> Personal Access Token (public_repo checked)
aragon_wiki_github_token needs to be set in TravisCI as a enviroment variable with the value. (Reminder do not show this in logs.)

This fixes Issue #107

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+language: python # Set the build language to Python
+
+python: 3.7.1 # Set the version of Python to use
+
+branches: master # Set the branch to build from
+
+install:
+    - pip install mkdocs # Install the required dependencies
+    - pip install mkdocs-material # Install the required dependencies
```
Comment (sohkai) [inline .travis.yml]: We can put both dependencies on the same line

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  target-branch: gh-pages
+  verbose: true
+  on:
+    branch: master
+
+matrix:
+  include:
+    - python: 3.7
+      dist: xenial
+      sudo: true
```
Comment (sohkai) [inline .travis.yml]: Do we need to specify the distribution and use sudo? Could we get away with not including the matrix?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+script:
+    - mkdocs build --verbose --clean # Build a local version of the docs
+
+deploy:
+  provider: pages
+  skip-cleanup: true
+# github-token is Generated in Github Developer Settings -> Personal Access Token (public_repo checked)
+# It also needs to be set in TravisCI as a enviroment variable with the value. (Reminder do not show this in logs.)
+  github-token: $aragon_wiki_github_token
```
Comment (sohkai) [inline .travis.yml]: Let's rename this `GH_TOKEN`, since that's what we use in a few other repos.

File: .travis.yml (hunk 2 above)
Comment (antonyip) [reply .travis.yml]: This error comes up when i do not include the matrix on the TravisCI log.

3.7.1 is not installed; attempting download
Downloading archive: https://travis-ci-python-archives.global.ssl.fastly.net/binaries/ubuntu/14.04/x86_64/python-3.7.1.tar.bz2
0.20s$ curl -sSf -o python-3.7.1.tar.bz2 ${archive_url}
curl: (22) The requested URL returned error: 403 Forbidden
Unable to download 3.7.1 archive. The archive may not exist. Please consider a different version.

Tried on Travis-CI.com and Travis-CI.org.

Will try and see if the build works with python 3.6

Comment (antonyip) [reply .travis.yml]: This is fixed now to use python3.6

**Comment by sohkai** [comment]
Looks good, thanks @antonyip!

@Smokyish I've turned on Travis for the wiki now; if you want we can merge this and see if it publishes a new version correctly from the master branch.

**Comment by Smokyish** [comment]
Thanks @antonyip & @sohkai, merging now!

**Comment by sohkai** [comment]
Hmm, perhaps the wiki is too big for travis to clone completely 😅. It looks like it's timing out: https://travis-ci.org/aragon/aragon-wiki/builds/506359501.

@antonyip Do you want to take a stab at fixing this? We may want something like `travis_wait` or figure out how to clone more quickly.

**Comment by antonyip** [comment]
Sure, I'm up for it. I think I will need access to your TravisCI tho...

**Pull request merged**